lib/calls.tsx

```diff
@@ -12,23 +12,40 @@ export async function getAllExperiences() {
 
 export async function getFilteredExperinces(query: string) {
     const allExperiences = await axios.get(`http://localhost:3000/api/getPosts`, {
-        params : {
+        params: {
             isDistinct: false,
         }
     });
 
+    const highlightQuery = (text: string, query: string) => {
+        const highlightedText = text.replace(new RegExp(query, 'gi'), (match) => `<mark>${match}</mark>`);
+        return highlightedText;
+    };
+
     // filter per query
-    const filterdExperiences = allExperiences.data.filter((exp: any) => exp.description?.includes(query));
+    // const filterdExperiences = allExperiences.data.filter((exp: any) => (exp.description?.toLowerCase().includes(query) | exp.title?.toLowerCase().includes(query)));
+    const filteredExperiences = allExperiences.data.filter((exp: any) => {
+        const queryLower = query.toLowerCase();
```

Comment:
I was wondering about this. I think there should be a file called `libs/normalize/ts` where we specify how we will trim whitespace and make text case insensitive before adding to the database